File: Codeforces/Golang/800/785A - Anton and Polyhedrons/main.go
package main

import (
	"fmt"
)

func main() {
	var n, sum int
	fmt.Scan(&n)
	for i := 0; i < n; i++ {
		var s string
		fmt.Scan(&s)
		switch s {
		case "Tetrahedron":
			sum += 4
		case "Cube":
			sum += 6
		case "Octahedron":
			sum += 8
		case "Dodecahedron":
			sum += 12
		case "Icosahedron":
			sum += 20
		}
	}
	fmt.Println(sum)
}

File: Codeforces/Rust/800/785A - Anton and Polyhedrons/src/main.rs
use std::io;

fn main() {
    let mut input = String::new();
    let _ = io::stdin().read_line(&mut input);

    let n: i64 = input.trim().parse().unwrap();
    let mut sum = 0;

    for _ in 0..n {
        let mut s = String::new();
        let _ = io::stdin().read_line(&mut s);

        match s.trim() {
            "Tetrahedron" => sum += 4,
            "Cube" => sum += 6,
            "Octahedron" => sum += 8,
            "Dodecahedron" => sum += 12,
            "Icosahedron" => sum += 20,

            _ => (),
        }
    }
    println!("{}", sum)
}
